superduper/components/template.py

```diff
@@ -46,6 +46,7 @@ def __post_init__(self, db, artifacts, substitutions):
             )
         if self.template_variables is None:
             self.template_variables = self.template.variables
+        logging.info(f"Template variables: {self.template_variables}")
```

Comment:
if u can print vertically its more readable

Reply:
I tested it, and directly printing it makes it easier to view.